examples/pytorch/graphsage/experimental/README.md

```diff
@@ -39,11 +44,15 @@ python3 train_dist.py --server --graph-name ogb-product --id 2 --num-client 4 --
 python3 train_dist.py --server --graph-name ogb-product --id 3 --num-client 4 --conf_path data/ogb-product.json --ip_config ip_config.txt
 ```
 
+When testing in the standalone mode, we don't need to run the servers.
```

Comment:
I'm a little confused here. Why we don't need server here? In my opinion, the standalone mode is just distributed training with one machine.

Reply:
the server is launched with the launch script afterwards. the standalone mode is to allow users to run their code without launch script. For example, running it in Jupyter Notebook environment.